source/blog/2018-11-19-ember-3-6-released.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Ember 3.6 Released
+author: Melanie Sumner, Kenneth Larsen, Chris Garrett
+tags: Releases, 2018, 3, 3.6
+responsive: true
+---
+
+Today the Ember project is releasing version 3.6 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.6
+Ember.js 3.6 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is 2 new feature, 6 deprecations, and 13 bugfixes in this version.
+
+#### New Features (2)
+
+**Native Classes (1 of 2)**
+
+The [ES Class](https://emberjs.github.io/rfcs/0240-es-classes.html) and [Native Class Constructor Update](https://emberjs.github.io/rfcs/0337-native-class-constructor-update.html) RFCs officially ship in 3.6! This feature means that usage of native classes has stabilized, and they are considered public API whose behavior will not change.
+
+**Great, so I can use `class` syntax now?!**
+Hold your horses! It's not _quite_ that simple yet.
+
+The behavior of native classes is stable, but currently Ember does not support or recommend the usage of class fields or decorators, which are both still undergoing the TC39 process. Subsequent RFCs will have to be made to make these officially part of Ember.
```

Comment:
I think we should give 1 quick sentence on TC39, to be clear that it’s not some special Ember thing